### PR TITLE
Adds skrell tees and dress shirts for casual style.

### PR DESCRIPTION
**Pull request by NonQueueingMatt**
As per title. Here are a few examples.

immagine
immagine
immagine

### CODE REVIEW

**Comment by NonQueueingMatt** [comment]
!review

**Review by RyverStyx — APPROVED** [review]

File: code/modules/client/preference_setup/loadout/loadout_xeno/skrell.dm [hunk 1]
```diff
@@ -276,4 +276,40 @@ datum/gear_tweak/social_credit/tweak_item(var/obj/item/clothing/accessory/badge/
 	workcloak["ix cloak"] = /obj/item/clothing/accessory/poncho/shouldercape/cloak/ix
 	workcloak["oqi cloak"] = /obj/item/clothing/accessory/poncho/shouldercape/cloak/oqi
 	workcloak["iqi cloak"] = /obj/item/clothing/accessory/poncho/shouldercape/cloak/iqi
-	gear_tweaks += new /datum/gear_tweak/path(workcloak)
\ No newline at end of file
+	gear_tweaks += new /datum/gear_tweak/path(workcloak)
+
+/datum/gear/accessory/skrell/tees
+	display_name = "skrellian t-shirts"
```
Comment (Arrow768) [inline code/modules/client/preference_setup/loadout/loadout_xeno/skrell.dm:282]: Needs to be renamed here aswell

**Review by Arrow768 — DISMISSED** [review]

File: code/modules/client/preference_setup/loadout/loadout_xeno/skrell.dm (hunk 1 above)
Comment (NonQueueingMatt) [inline code/modules/client/preference_setup/loadout/loadout_xeno/skrell.dm:282]:
```suggestion
	display_name = "skrellian wetshirts"
```

File: code/modules/client/preference_setup/loadout/loadout_xeno/skrell.dm [hunk 2]
```diff
@@ -276,4 +276,40 @@ datum/gear_tweak/social_credit/tweak_item(var/obj/item/clothing/accessory/badge/
...
+	tees["pink eriuyushi nebula shirt"] = /obj/item/clothing/accessory/tshirt/skrell/nebula
+	tees["purple eriuyushi nebula shirt"] = /obj/item/clothing/accessory/tshirt/skrell/nebula/purple
+	tees["teal eriuyushi nebula shirt"] = /obj/item/clothing/accessory/tshirt/skrell/nebula/teal
+	tees["black eriuyushi nebula shirt"] = /obj/item/clothing/accessory/tshirt/skrell/nebula/black
+	tees["white eriuyushi nebula shirt"] = /obj/item/clothing/accessory/tshirt/skrell/nebula/white
+	tees["jargon eriuyushi nebula shirt"] = /obj/item/clothing/accessory/tshirt/skrell/nebula/jargon
+	gear_tweaks += new /datum/gear_tweak/path(tees)
+
+/datum/gear/accessory/skrell/dress
+	display_name = "skrellian dress shirts"
```
Comment (NonQueueingMatt) [inline code/modules/client/preference_setup/loadout/loadout_xeno/skrell.dm:303]:
```suggestion
	display_name = "skrellian formal wetshirts"
```

**Review by Arrow768 — APPROVED** [review]

**Review by Alberyk — APPROVED** [review]

**Pull request merged**